mls.bib

```diff
@@ -82,3 +82,10 @@ @Manual{Pouzet2006LucidSynchrone30
   year = {2006},
   url = {http://www.di.ens.fr/~pouzet/lucid-synchrone/},
 }
+
+@Manual{GettextManual,
+  author = {Drepper, Ulrich and Meyering, Jim and Pinard, François and Haible, Bruno},
+  title = {GNU gettext tools, version 0.21},
+  year = {2020},
+  url = {https://www.gnu.org/software/gettext/manual/gettext.pdf},
```

Comment:
```suggestion
  url = {https://www.gnu.org/software/gettext/manual/},
```
The decision was to keep it tied to the specific version, but don't deep-link the pdf-manual.

Since it is below 1.0 it may change completely in version 0.22, since we require file-compatibility that would be a mess. Or it may change very little. In the latter case we can just update the version we reference.